styles/signals.json

```diff
@@ -719,6 +719,38 @@
 			}],
 			"caption": "Ra 10 Rangierhalttafel / Verschubhalttafel"
 		},
+		{
+			"minzoom": 17,
+			"features" : [{
+				"type": "Point",
+				"coordinates": [50,50],
+				"properties": {
+					"railway":"signal",
+					"railway:signal:direction":"forward",
+					"railway:signal:wrong_road":"DE-ESO:db:zs6",
+					"railway:signal:wrong_road:form":"light"
+				}},
+				{
+				"type": "Point",
+				"coordinates": [50,50],
+				"properties": {
+					"railway":"signal",
+					"railway:signal:direction":"forward",
+					"railway:signal:wrong_road":"DE-ESO:dr:zs7",
+					"railway:signal:wrong_road:form":"light"
+				}},
+				{
+				"type": "Point",
+				"coordinates": [50,50],
+				"properties": {
+					"railway":"signal",
+					"railway:signal:direction":"forward",
+					"railway:signal:wrong_road":"DE-ESO:zs6",
```

Comment:
`DE-ESO:db:zs6` and `DE-ESO:dr:zs7` are suggested by the documentation and used by the JOSM preset but `DE-ESO:zs6` is not. That's why I suggest to change it to `DE-ESO:db:zs6` and skip the DR version of the sign because apart from 4 signs all Zs 6 signs are DB-like. The four remaining signs are mounted at Hl signals in Reuth (bei Plauen/Vogtland), Pölchow and Schwaan, i.e. their lifetime is limited.